interface/wx/zipstrm.h

```diff
@@ -476,14 +476,34 @@ class wxZipOutputStream : public wxArchiveOutputStream
 
         If the parent stream is passed as a pointer then the new filter stream
         takes ownership of it. If it is passed by reference then it does not.
+
+        UTF-8 Encoding will be used for filenames that are using non ASCII
+        characters.
+
+        @since 3.1.1
```

Comment:
I think this risks being misleading in the generated documentation, i.e. it would create an impression that it's the ctor itself which is new, which is not really the case as the code using it would still compile with older versions and would just work differently. It's probably better to just write in plain text that "Since versions 3.1.1, filenames in the generated archive will be encoded using UTF-8" and ideally also explain how can it be changed if really needed (although I have no idea why would anyone need to change it to be honest).

I also think it might be worth pretending that there is still only a single ctor (for each pointer/reference variant) instead of 2 overloaded ones, even if we don't actually have a default value for `conv` parameter, it would IMHO be simpler to understand if we still documented it like this, with `wxConvUTF8` being the default.